GA-worksheet.md

```diff
@@ -7,18 +7,17 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+A: To hold other tags, specifically for a website, sort of like a body tag.
```

Comment:
The header sits within the body (as opposed to the head, which is for metadata).

More specifically, the header tag actually holds the introductory material for the website, such as a logo and navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify).